### ISSUE TITLE
IOError: [Errno 2] No such file or directory: '/en_core_web_sm/en_core_web_sm-2.0.0a0/tagger/cfg'

### ISSUE THREAD

**Issue by irfan-zoefit** [explosion/spaCy#1221]
I'm trying to run the spacy-nightly in a new environment for testing purpose. After installation when I try to run the following code as mention in the example,
```python
import spacy
from spacy import displacy
try:
    unicode
except:
    unicode = str
text = unicode('But Google is starting from behind.')

nlp = spacy.load('en_core_web_sm')
doc = nlp(text)
displacy.serve(doc, style='ent')
```
I got the error:
```shell
IOError: [Errno 2] No such file or directory: '/home/irfan/PycharmProjects/nightly/.night/local/lib/python2.7/site-packages/en_core_web_sm/en_core_web_sm-2.0.0a0/tagger/cfg
```

## Info about spaCy

* **Python version:** 2.7.12
* **Platform:** Linux-4.4.0-87-generic-x86_64-with-Ubuntu-16.04-xenial
* **spaCy version:** 2.0.0a4

**Comment by nikeqiang** [comment]
Same problem.  Same setup but using python3.

**Comment by ines** [comment]
Ah, damn, this was a bug in introduced in the recent alpha update (`2.0.0a4`). Sorry about that – pushing a new update in a minute that fixes this.

In the meantime, `2.0.0a0` should work fine:

```bash
pip install spacy-nightly==2.0.0a0
```

**Comment by ines** [comment]
Fixed in the latest update, `2.0.0a6`!
